web/src/assets/locales/en.json

```
@@ -246,7 +246,16 @@
       "show_errors": "Show errors",
       "we_got_some_errors": "Oh no, we got some errors!",
       "duration": "Pipeline duration",
-      "created": "Created: {created}"
+      "created": "Created: {created}",
+      "debug": {
+        "title": "Debug",
+        "download_metadata": "Download Metadata File",
+        "error_fetching": "Error fetching metadata",
```

To have consistent keys:

```suggestion
        "metadata_download_error": "Error downloading metadata",
```